ChatSecure/Classes/View Controllers/OTRComposeGroupViewController.swift

```diff
@@ -224,7 +224,7 @@ open class OTRComposeGroupViewController: UIViewController, UICollectionViewDele
                     for row in 0..<mappings.numberOfItems(inSection: section) {
                         var buddy:OTRXMPPBuddy? = nil
                         if let roomOccupant = viewHandler.object(IndexPath(row: Int(row), section: Int(section))) as? OTRXMPPRoomOccupant {
-                            OTRDatabaseManager.shared.uiConnection?.read({ (transaction) in
+                            OTRDatabaseManager.shared.readConnection?.read({ (transaction) in
```

Comment:
Good catch. Although it's not enforced, the `uiConnection` shouldn't be used off the main thread.

Reply:
Was causing YapDatabaseConnection -> readWithBlock -> NonMainThreadException for me